docker/cloudbuild.yaml

```diff
@@ -24,26 +24,40 @@ steps:
     # TODO(byk): We may also build this part as a builder image everytime
     #            there's a push to onpremise and use that image for
     curl -L "https://github.com/getsentry/onpremise/archive/v10.tar.gz" | tar xzf - --strip-components=1
-    ./install.sh
-    docker-compose run --rm web createuser --superuser --email test@example.com --password test123TEST
     # The following trick is from https://stackoverflow.com/a/52400857/90297 with great gratuity
```

Comment:
Moving this trick above prevents the recreation of containers after the installation. Ideally, it'd be a separate PR but since it is small, I snucked it in.